tests/cuckoo/stress.rs

```diff
@@ -102,7 +105,10 @@ fn stress_delete_thread(env: Arc<Environment>) {
     while !env.finished.load(Ordering::SeqCst) {
         let idx = env.ind_dist.sample(&mut rng);
         let in_use = env.in_use.lock();
-        if (*in_use)[idx].compare_and_swap(false, true, Ordering::SeqCst) {
+        if (*in_use)[idx]
+            .compare_exchange_weak(false, true, Ordering::SeqCst, Ordering::SeqCst)
```

Comment:
Why did you make this use `_weak`?
```suggestion
            .compare_exchange(false, true, Ordering::SeqCst, Ordering::Relaxed)
```